packages/crypto-wallet-core/package.json

```diff
@@ -21,9 +21,16 @@
   "author": "Micah Riggan",
   "license": "MIT",
   "dependencies": {
+    "base-x": "^3.0.7",
+    "bcryptjs": "2.4.3",
     "bitcore-lib": "^8.8.4",
     "bitcore-lib-cash": "^8.8.4",
     "ethers": "4.0.37",
+    "elliptic": "6.4.1",
```

Comment:
it seems like the package.json file didn't get merged correctly, shouldn't see any packages here that you didn't add